gensim/scripts/word2vec2tensor.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Loreto Parisi <loretoparisi@gmail.com>
+# Copyright (C) 2016 Silvio Ogliastri <silvio.olivastri@gmail.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
```

Comment:
May I ask for joint copyright as in https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#legal ?